src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperDirectiveSpanVisitor.cs

```diff
@@ -82,50 +83,42 @@ public IEnumerable<TagHelperDescriptor> GetDescriptors(Block root)
 
         public override void VisitSpan(Span span)
```

Comment:
All of the changes in this method are a reaction to the new `TagHelper` directive `ChunkGenerator` formats of not holding onto the `LookupText`. Was able to significantly simplify the code in this method as a result. In the end does the same thing with the exception of `.LookupText` => `GetDirectiveText(span)`.